app/scripts/HiGlassComponent.js

```diff
@@ -230,6 +230,7 @@ class HiGlassComponent extends React.Component {
     this.zooming = false;
 
     // Bounded functions
```

Comment:
I know this isn't part of the PR, but should this comment actually say "Bound functions"?